CMakeLists.txt

```diff
@@ -63,6 +63,9 @@ else()
   message( STATUS "There is no ConfTargets.cmake" )
 endif()
 
+set( MFX_CONFIG_FILE ${BUILDER_ROOT}/profiles/fastboot.cmake )
```

Comment:
no, no, no. Fastboot will _not_ be a default config! never!

Please, cherry-pick this patch: https://github.com/dvrogozh/MediaSDK/commit/01c44ad8ad50bba79294bfda22a20c3a8b899d70. This is a starting point! What you are doing is simply incorrect. In my original patch user can either add -DMFX_ENABLE_H264_VIDEO_DECODER=OFF options or have -DMFX_CONFIG_FILE=/path/to/file. Try it works. And this actually should be put into the commit message.